### PR TITLE
Adding a description icon to cards when they contain a description without any checkmarks

### PR DESCRIPTION
**Pull request by MonkeySon**
* Resolves: #1873 
* Target version: master 

### Summary

Adding a description icon to cards when they contain a description without any checkmarks.

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is **not required**

### CODE REVIEW

File: src/components/cards/CardBadges.vue [hunk 1]
```diff
@@ -27,6 +27,7 @@
 		<div v-if="card.description && checkListCount > 0" class="card-tasks icon icon-checkmark">
 			{{ checkListCheckedCount }}/{{ checkListCount }}
 		</div>
+		<div v-else-if="card.description && checkListCount == 0" class="icon icon-description" />
```
Comment (juliushaertl) [inline src/components/cards/CardBadges.vue:30]:
```suggestion
		<div v-else-if="card.description.trim() && checkListCount == 0" class="icon icon-description" />
```

To make sure that a description only containing spaces is not showing up.

**Review by juliushaertl — CHANGES_REQUESTED** [review]
Thanks for your pull request. I've added a minor comment, otherwise :+1:

**Comment by juliushaertl** [comment]
Mind to rebase and squash your comments? Then the DCO check should also be happy without the merge commit :)

**Comment by MonkeySon** [comment]
Sorry I am rather new to git pull requests, rebases and squashes.
If I understand you correctly, I should merge (= squash)  the last two commits (Merge & "Ignore descriptions that only contain spaces")?

**Comment by MonkeySon** [comment]
Ok, I added the DOC to the latest commit, but I don't know how to get rid of the empty merge (master -> add-description-icon) commit.
I tried the command `git rebase -i HEAD~2`, but it does not include merge commits.

**Review by juliushaertl — APPROVED** [review]

**Comment by juliushaertl** [comment]
>  Now, after I manually signed-off the last commit, everything should be conform. Am I right with this assumption?

Yes, that is all fine now. Thanks a lot for your pull request :)

**Pull request merged**